### PR TITLE
Update colors with gray 750

### PR DESCRIPTION
**Pull request by araiford**
Adds Gray 750 to colors page
Also adds an HR rule to CSS as an option to improve spacing in our docs.

### CODE REVIEW

File: _docs/style/colors.md [hunk 1]
```diff
@@ -66,25 +62,28 @@ illustrations. They should never overpower the primary colors.
...
+  {% include chips.html class="hxCol-xs-12 hxCol-md-3 color-chip" bg-color="#e0e0e0" color="#616161" csscode="Gray 800 on Gray 300" hex="#616161 on #E0E0E0" ratio="4.7 : 1" %}
+  {% include chips.html class="hxCol-xs-12 hxCol-md-3 color-chip" bg-color="#d8d8d8" color="#424242" csscode="Gray 900 on Gray 400" hex="#424242 on #D8D8D8" ratio="7.1 : 1" %}
+  {% include chips.html class="hxCol-xs-12 hxCol-md-3 color-chip" bg-color="#BDBDBD" color="#424242" csscode="Gray 900 on Gray 500" hex="#424242 on #BDBDBD" ratio="5.4 : 1" %}
+  {% include chips.html class="hxCol-xs-12 hxCol-md-3 color-chip" bg-color="#9E9E9E" color="#333333" csscode="Gray 950 on Gray 600" hex="#333333 on #9E9E9E" ratio="4.7 : 1" %}
+  {% include chips.html class="hxCol-xs-12 hxCol-md-3 color-chip" bg-color="#757575" color="#000000" csscode="Gray 1000 on Gray 700" hex="#000000 on #757575" ratio="4.6 : 1" %}
+  {% include chips.html class="hxCol-xs-12 hxCol-md-3 color-chip" bg-color="#6b6b6b" color="#eeeeee" csscode="Gray 100 on Gray 750" hex="#eeeeee on #6b6b6b" ratio="4.6 : 1" %}
```
Comment (technabors) [inline _docs/style/colors.md]: Lets capitalize the HEX values for consistencies sake.

File: _docs/style/colors.md [hunk 2]
```diff
@@ -3,22 +3,17 @@ title: Color Palette
...
 
-The use of color establishes cohesion across Rackspace products. Our color
-palette has been selected to enhance usability while adhering to accessibility
-criteria. Guidelines for color usage are outlined for
-[typography](/style/typography/) and individual components. Avoid adding color
-to "spice things up" as this will distract the user from his or her workflow.
 
-# Primary Colors
+
+### Primary Colors
```
Comment (nimbinatus) [inline _docs/style/colors.md]: Any reason to start with L3 vs L2?

Comment (araiford) [reply _docs/style/colors.md]: @nimbinatus -- yep!  The page already has an H1 and H2.

**Comment by araiford** [comment]
@nimbinatus -- I fixed conflicts on the Add Gray 750 PR

**Pull request merged**